package.json

```diff
@@ -33,10 +34,20 @@
             {
                 "command": "ucl-rsdg.helloWorld",
                 "title": "This is the title: Hello World",
+                "category": "Oracc",
                 "icon": {
                     "light": "media/preview_light.svg",
                     "dark": "media/preview_dark.svg"
                 }
+            },
+            {
+                "command": "ucl-rsdg.validationButton",
```

Comment:
I think a name like `ucl-rsdg.validate` or `ucl-rsdg.validateAtf` would be slightly better, because it doesn't tie it to the button directly (we may want to call the same command a different way, such as through a menu item, or through a keyboard shortcut as we already do). But we need to make sure we change the name everywhere!